version/version.go

```diff
@@ -39,7 +39,11 @@ func VerifyExchangeVersion1(exch_version string, checkWithPreferred bool) error
 	} else if comp, err := semanticversion.CompareVersions(exch_version, version_for_check); err != nil {
 		return fmt.Errorf("Failed to compare the versions. %v", err)
 	} else if comp < 0 {
-		return fmt.Errorf("The current exchange version %v does not meet the requirement. The required version is %v or above. Please upgrade the exchange.", exch_version, version_for_check)
+		return fmt.Errorf("The current exchange version %v does not meet the requirement. The required version is %v or above. "+
+			"Please upgrade the exchange.", exch_version, version_for_check)
+	} else if checkExactVersion && comp > 0 {
```

Comment:
This parts needs more discussion. Please start a slack group discussion with @dabooz, @TheMosquito and myself.